src/sentry/testutils/cases.py

```diff
@@ -844,9 +848,23 @@ def assertDialogSuccess(self, resp):
 class SnubaTestCase(TestCase):
     def setUp(self):
         super(SnubaTestCase, self).setUp()
-
+        self.snuba_eventstream = SnubaEventStream()
+        self.snuba_tagstore = SnubaCompatibilityTagStorage()
         assert requests.post(settings.SENTRY_SNUBA + '/tests/drop').status_code == 200
 
+    def store_event(self, *args, **kwargs):
+        with contextlib.nested(
+            mock.patch('sentry.eventstream.insert',
+                       self.snuba_eventstream.insert),
+            mock.patch('sentry.tagstore.delay_index_event_tags',
```

Comment:
cc @alex-hofsteede: Here we need to swap out `SnubaTagStorage` with `SnubaCompatibilityTagStorage` because we want to go through (more of) the real event pipeline.

Reply:
Does this cause any issues? `SnubaCompatibilityTagStorage` is the `SENTRY_TAGSTORE` being used in production, so it seems fine to use it here too.

Reply:
I was just wondering if there was a reason we couldnt set the global setting to the same value instead of `SnubaTagStorage`

Reply:
nope, there shouldn't be any problem with using `SnubaCompatibilityTagStorage` everywhere.

Reply:
kk. I think this is fine for now, but if I find tests that indirectly instrument those methods I'll revisit.